motion/project/template/ios/files/readme.md

```diff
@@ -60,56 +60,43 @@ app.entitlements['beta-reports-active'] = true
 
 As of iOS 11, Apple requires the use of Asset Catalogs for defining
 icons and launch screens. You'll find icon and launch screen templates
-under `./resources/Assets.xcassets`. You can run the following script
-to generate all the icon sizes (once you've specified `1024x1024.png`).
-Keep in mind that your `.png` files _cannot_ contain alpha channels.
+under `./resources/Assets.xcassets`. 
 
-Save this following script to `./gen-icons.sh` and run it:
+The current build has a built-in icon generate that can be triggered 
+from your Rakefile. The first step is to create a PNG file (keep in 
+mind that your `.png` file _cannot_ contain alpha channels) and place 
+it in your `resources` directory. This file should be a minimum of 1024 
+x 1024 pixels (it can be higher) and should be square (to prevent 
+aspect distortion during the generation process).
+
+To generate your icon in the Asset catalogue, add a dependency to your `Rakefile` as follows:
+
+```ruby
+task :icons => 'resources/app-icon.icon_asset'
```

Comment:
Is this supposed to be  just `icons` or is it `build:icons` ?

And should it say `add or uncomment` because it'll be there, commented out, for new projects